docs/src/app/components/pages/components/CircularProgress/ExampleDeterminate.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import CircularProgress from 'material-ui/lib/circular-progress';
+
+export default class CircularProgressExampleDeterminate extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      completed: 0,
+    };
+  }
+
+  componentDidMount() {
+    this.timer = setTimeout(() => this._progress(5), 1000);
+  }
+
+  componentWillUnmount() {
+    clearTimeout(this.timer);
+  }
+
+  _progress(completed) {
```

Comment:
I think that the `_` is not needed. The component method are supposed to be private by default no?